app/shared/video-player/src/commonMain/kotlin/ui/progress/MediaProgressSlider.kt

```diff
@@ -227,13 +231,17 @@ fun MediaProgressSlider(
                 )
             }
 
-            Canvas(Modifier.align(Alignment.CenterEnd).padding(end = 4.dp)) {
-                // draw stop
-                drawCircle(
-                    stopColor,
-                    radius = 2.dp.toPx(),
-                    blendMode = BlendMode.Src, // override background
-                )
+            Canvas(Modifier.matchParentSize()) {
+                if (state.totalDurationMillis == 0L) return@Canvas
+                chapters.forEach {
```

Comment:
```suggestion
                for (index in 1..chapters.lastIndex) { 
                    val chapter = chapters[index]
```

需啊跳过第一个章节, 否则会导致进度条的最开始一定会有一个章节标记

Reply:
不会有，如果有就是视频开头就是op
<img width="1280" alt="Snipaste_2024-08-04_14-02-28" src="https://github.com/user-attachments/assets/f79d488b-ad51-43de-bf85-30d1186118c8">